extensions/amp-brightcove/amp-brightcove.md

```diff
@@ -0,0 +1,62 @@
+<!---
+Copyright 2015 Brightcove. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-brightcove"></a> `amp-brightcove`
```

Comment:
Could you please add a link to your extension here? https://github.com/ampproject/amphtml/blob/8e918dd48c7cee2de7b362633ea6b8dcba845e5a/extensions/README.md